libs/ledger-live-common/src/exchange/swap/hooks/useSwapTransaction.ts

```diff
@@ -86,7 +120,9 @@ export const useSwapTransaction = ({
   const { account: toAccount } = toState;
   const transaction = bridgeTransaction?.transaction;
 
-  const fromAmountError = useFromAmountError(bridgeTransaction.status.errors);
+  const fromAmountError = useFromAmountStatusMessage(bridgeTransaction, ["amount"]);
+  // treat the gasPrice error as a warning for swap.
+  const fromAmountWarning = useFromAmountStatusMessage(bridgeTransaction, ["gasPrice"]);
```

Comment:
This is only a suggestion, if I understood correctly we can not have both of them at the same time, I wonder if it will be simpler to read use only "fromAmountError" and add a "type" and "error" attribute there...